src/OnlineRetailPortal.Services/Services/WriterHelper.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+
+namespace OnlineRetailPortal.Services.Services
+{
+    class WriterHelper
+    {
+
+        public enum ImageFormat
+        {
+            bmp,
+            jpeg,
+            gif,
+            tiff,
+            png,
+            unknown
+        }
+        /// <summary>
+        /// takes the file as a byte[] and returns the type of image, returns Unkown if other unrecognized
+        /// </summary>
+        /// <param name="bytes"></param>
+        /// <returns></returns>
+        public static ImageFormat GetImageFormat(byte[] bytes)
+        {
+            var bmp = Encoding.ASCII.GetBytes("BM");     // BMP
+            var gif = Encoding.ASCII.GetBytes("GIF");    // GIF
+            var png = new byte[] { 137, 80, 78, 71 };              // PNG
+            var tiff = new byte[] { 73, 73, 42 };                  // TIFF
+            var tiff2 = new byte[] { 77, 77, 42 };                 // TIFF
+            var jpeg = new byte[] { 255, 216, 255, 224 };          // jpeg
+            var jpeg2 = new byte[] { 255, 216, 255, 225 };         // jpeg canon
```

Comment:
rename this to jpegCannon . same with tiff2 variable

Reply:
what do we do when image format is `unknown` ??

Reply:
If the image format is "unknown" we return a 400 response with an "invalid image " message.